### PR TITLE
Add windows support for jsonapi configure

### PR DESCRIPTION
**Pull request by Project0**
I am still unsure about handling the gopass_native_host.exe binary. 
I tried first the windows symlink approach (mklink), but this requires higher privileges in windows (https://groups.google.com/forum/#!topic/golang-nuts/shRlhV3Fe2E). 

This means also the global install support does not work without that -> user needs to run gopass (or cmd) explicitly with higher privileges (run as admin). 
While this is ok for global, i decided it should be possible to configure the jsonapi for user only without admin rights.

For now it means, if there are new requirements or changes in the api for the web extension the user needs to re-run the configure to update native binary.

### CODE REVIEW

**Comment by Project0** [comment]
#899

**Comment by Project0** [comment]
ready to review/merge

**Comment by Project0** [comment]
actually i added the GOOS hack only once a time ;-), but let me check how much we can improve here

File: pkg/action/jsonapi.go [hunk 1]
```diff
@@ -119,12 +54,12 @@ func (s *Action) getLibPath(ctx context.Context, c *cli.Context, browser string,
 	return c.String("libpath"), nil
 }
 
-func (s *Action) getWrapperPath(ctx context.Context, c *cli.Context) (string, error) {
+func (s *Action) getWrapperPath(ctx context.Context, c *cli.Context, defaultWrapperPath string, wrapperName string) (string, error) {
```
Comment (dominikschulz) [inline pkg/action/jsonapi.go]: You could change this to `defaultWrapperPath, wrapperName string`.

**Comment by colemickens** [comment]
Where does the binary live? Gopass is not detecting any of my browsers and I'm trying to configure jsonapi manually. Thanks.

**Comment by colemickens** [comment]
Just found this. Cool trick, I was actually thinking of implementing it since I missed that jsonapi had Windows support.

>Windows: The jsonapi setup copies the current gopass binary as wrapper ( gopass_native_host.exe calls directly the listener).

**Pull request merged**